sentry/src/main/java/io/sentry/metrics/MetricsApi.java

```diff
@@ -554,8 +558,20 @@ public void timing(
     final @Nullable LocalMetricsAggregator localMetricsAggregator =
         aggregator.getLocalMetricsAggregator();
 
-    aggregator
-        .getMetricsAggregator()
-        .timing(key, callback, durationUnit, enrichedTags, stackLevel, localMetricsAggregator);
+    final @Nullable ISpan span = aggregator.startSpanForMetric("metric.timing", key);
```

Comment:
```suggestion
    final @Nullable ISpan span = aggregator.startSpanForMetric("metrics.timing", key);
```
should it be metric or metrics?

Reply:
That's fine this way, as it matches [python](https://github.com/getsentry/sentry-python/blob/0901953c93071e858f4da67c1e864766ae19c002/sentry_sdk/metrics.py#L800C48-L800C61) / [ruby](https://github.com/getsentry/sentry-ruby/pull/2255/files#diff-44d204371288f9a84dd2ed1c054a2b09d85388267ba249b83ca08ea251b1e6f9R17)